docs/packages/README.md

```diff
@@ -1,30 +1,63 @@
 # Package Development
 
-A package is a unit added to your application for enhancement which includes routes, controllers, views, and configuration specifically. Packages are created to manage your large applications into smaller units.  
-In the bagisto, we have created plenty of packages at path `packages/Webkul/`. You can find a basic tree-structure of package below:
+A package is a unit added to your application for enhancement which includes routes, controllers, views, and configuration specifically. Packages are created to manage your large applications into smaller units.
 
-```php
-- module-name/
+In the Bagisto, we have created plenty of packages at path `packages/Webkul/`. You can find a basic tree-structure of the package below:
```

Comment:
- In the Bagisto, we have created plenty of packages at path `packages/Webkul/`. You can find a basic tree-structure of the package below:

Should be: 

In Bagisto, we have created plenty of packages inside `packages/Webkul/`. You can find a basic tree-structure of the package below:

Reply:
Done!